### PR TITLE
feat(FEC-8868): mobile ads preset

### PR DESCRIPTION
**Pull request by odedhutzler**
Description of the Changes

Added fullscreen and volume controls to the IMA mobile preset.
moved everything to the left, so it won't clash with IMA's skip button.
Render the default player view also on mobile

References:
https://github.com/kaltura/playkit-js-ui/pull/334

unknown

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment 
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/ui-presets/ads.js [hunk 1]
```diff
@@ -24,17 +24,29 @@ import {PlaybackControls} from '../components/playback-controls';
 export function adsUI(props: any): ?React$Element<any> {
   if (useDefaultAdsUi(props)) {
     return (
-      <div className={style.adGuiWrapper}>
+      <div className={[style.adGuiWrapper, style.onTopAd].join(' ')}>
+        <KeyboardControl player={props.player} config={props.config} />
```
Comment (yairans) [inline src/ui-presets/ads.js]: verify we want this. as now the user can seek to the end using the keyboard

File: src/components/bottom-bar/_bottom-bar.scss [hunk 2]
```diff
@@ -53,6 +53,10 @@
   visibility: hidden;
 }
 
+.player .on-top-ad .bottom-bar{
+  width: auto;
```
Comment (yairans) [inline src/components/bottom-bar/_bottom-bar.scss]: check if can be added to `adGuiWrapper` class

File: src/ui-presets/ads.js [hunk 3]
```diff
@@ -24,17 +24,29 @@ import {PlaybackControls} from '../components/playback-controls';
...
+          </div>
+          <PlaybackControls player={props.player} />
+          <BottomBar>
             <div className={style.leftControls}>
-              <AdNotice />
+              <PlaybackControls player={props.player} />
+              <TimeDisplayAdsContainer />
             </div>
-          </TopBar>
+            <div className={style.leftControls}>
```
Comment (yairans) [inline src/ui-presets/ads.js]: duplicate

File: src/ui-presets/ads.js [hunk 4]
```diff
@@ -89,10 +101,9 @@ function getAdsUiCustomization(): Object {
  */
 function useDefaultAdsUi(props: any): boolean {
   try {
-    let isMobile = !!props.player.env.device.type;
     let adsRenderingSettings = props.player.config.plugins.ima.adsRenderingSettings;
     let useStyledLinearAds = adsRenderingSettings && adsRenderingSettings.useStyledLinearAds;
-    return isMobile || useStyledLinearAds;
+    return useStyledLinearAds;
```
Comment (yairans) [inline src/ui-presets/ads.js]: verify `isMobile` is redundant

Comment (odedhutzler) [reply src/ui-presets/ads.js]: verified

File: src/ui-presets/ads.js (hunk 1 above)
Comment (odedhutzler) [reply src/ui-presets/ads.js]: i think we do as we have it in the other ads view as well (and he can't really skip the ad)

File: src/components/bottom-bar/_bottom-bar.scss (hunk 2 above)
Comment (odedhutzler) [reply src/components/bottom-bar/_bottom-bar.scss]: cannot

File: src/ui-presets/ads.js (hunk 3 above)
Comment (odedhutzler) [reply src/ui-presets/ads.js]: removed, thanks!

**Pull request merged**